xbmc/network/WakeOnAccess.cpp

```diff
@@ -66,6 +71,118 @@ static unsigned long HostToIP(const std::string& host)
   return inet_addr(ip.c_str());
 }
 
+#define LOCALIZED(id) g_localizeStrings.Get(id)
+
+static void ShowDiscoveryMessage(const char* function, const char* server_name, bool new_entry)
+{
+  std::string message, heading = LOCALIZED(13033);
+
+  if (new_entry)
+  {
+    CLog::Log(LOGINFO, "%s - Create new entry for host '%s'", function, server_name);
+    message = StringUtils::Format(LOCALIZED(13035).c_str(), server_name);
+  }
+  else
+  {
+    CLog::Log(LOGINFO, "%s - Update existing entry for host '%s'", function, server_name);
+    message = StringUtils::Format(LOCALIZED(13034).c_str(), server_name);
+  }
+  CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Info, heading, message, 4000, true, 3000);
```

Comment:
Yes it's relatively new. Look for `CEventLog::GetInstance()` in the existing code base. I added a few usages. You can either just create an event for the event log or also tell it to show a kai toast notification.

But I wouldn't let this block this PR, it was just an idea.